uint8 SensorID
float64 Data
